recipes/stata_kernel/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "stata_kernel" %}
+{% set version = "1.12.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stata_kernel-{{ version }}.tar.gz
+  sha256: 89885bd85f3debb62ee993e58f8108cb931a22d95885327cb53958914ed0bc7b
+
+build:
+  number: 0
+  skip: true   # [py2k]
```

Comment:
This is the default on conda-forge and doesn't need to be done here.

Reply:
With the `skip` gone you can add `noarch: python` and this will drastically reduce the build load from this package.

Reply:
Thanks, done.

Reply:
> noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.

Do I get this linter warning because of the extra dependency on windows and/or the platform-dependent test commands? If so, it might not be a true noarch package after all.

Reply:
Crap, I missed the `pywin32` dependency--my bad. Yeah, this package can't be noarch after all.